tests/gpt_j.rs

```diff
@@ -0,0 +1,89 @@
+use rust_bert::gpt_j::{
```

Comment:
Can you please add this test to the integration tests as well (after https://github.com/guillaume-be/rust-bert/blob/78da0f4814da00139551b615d6d06e7e92f59a79/.github/workflows/continuous-integration.yml#L117)

Reply:
Done